linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/VarGen.scala

```diff
@@ -132,15 +132,60 @@ private[emitter] final class VarGen(jsGen: JSGen, nameGen: NameGen,
     }
   }
 
+  /** Call what is stored in the given global var via dynamic import. */
+  def callDynamicGlobalVar[T: Scope](field: String, scope: T, args: List[Tree])(
```

Comment:
Not super happy about having to bake the call into here. Unsure what a better option is.